get_all_manifests.sh

```diff
@@ -156,10 +157,13 @@ download_manifest() {
         return 1
     fi
 
-    mkdir -p ./opt/manifests/${target_path}
-    cp -rf ${repo_dir}/${source_path}/* ./opt/manifests/${target_path}
+    mkdir -p ${DST_MANIFESTS_DIR}/${target_path}
+    cp -rf ${repo_dir}/${source_path}/* ${DST_MANIFESTS_DIR}/${target_path}
 }
 
+# Cleanup existing manifests
+rm -rf ${DST_MANIFESTS_DIR}/*
```

Comment:
on the one hand i understand, if some old manifests are removed from components, we should remove them as well by "make"
on the other hand if i have create some local files in the folder, they will get removed if i run "make"
what i normally do is "make clean get-manifests" 
maybe we should ask others aware if we wanna have the force "rm" before download new ones?